docs/documentation/Phonons/Castep_Phonons/Introduction.md

```diff
@@ -0,0 +1,315 @@
+
+This guide introduces the concepts, keywords and techniques needed to
+set up and run CASTEP calculations of lattice dynamics and vibrational
+spectroscopy. The material covers the various lattice-dynamical and
+related methods implemented in CASTEP, how to set up a calculation, and
+presents simple examples of the major types of calculation. It is
+assumed that the reader is familiar with the general CASTEP input and
+output files and keywords to the level of, for example the tutorials on
+geometry optimisation which may be found at
+[www.castep.org](http://www.castep.org). It describes how to analyse the
```

Comment:
Should at least point to the docs site. Perhaps the geom-opt documentation pages (until someone write a geom-opt tutorial)